yffi/src/lib.rs

```diff
@@ -2562,7 +2571,7 @@ impl Prelim for YInput {
                             .to_str()
                             .unwrap()
                             .to_owned();
-                        let value = values.offset(i).read().into();
```

Comment:
Apparently the issue lied here: when we try to insert prelim map filled with initial content, we tried to forcefully convert all nested values onto Any type, even if they were prelim types themselves.